pkg/plugin/work-in-progress/event_handler.go

```diff
@@ -40,6 +38,7 @@ type GitHubWIPPRHandler struct {
 
 var (
 	handledPrActions = []string{"opened", "reopened", "edited"}
+	defaultPrefixes  = []string{"WIP", "DO NOT MERGE", "DON'T MERGE", "WORK-IN-PROGRESS"}
```

Comment:
@hemanik Are we missing `WORK IN PROGRESS` here?

Reply:
according to the requested changes in the [issue](https://github.com/arquillian/ike-prow-plugins/issues/28#issuecomment-382414676) no. According to common sense, we could miss it, but I would say wait for some time to get the idea which patterns (prefixes) are used in the projects. After all, it is still configurable

Reply:
@MatousJobanek I totally understand your point of this is configurable. But I think `Work In Progress` is common pattern which you can see in general

Reply:
Personally, I use only `WIP` or `do not merge`. But that is my opinion. 
If you are used to using this prefix as well, then feel free to open a PR with this change ;-)